modules/migrating-sccs-during-the-manual-upgrade.adoc

```diff
@@ -0,0 +1,360 @@
+// Module included in the following assemblies:
+//
+// * upgrading/upgrade-roxctl.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="migrating-sccs-during-the-manual-upgrade_{context}"]
+= Migrating SCCs during the manual upgrade
+
+By migrating the security context constraints (SCCs) during the manual upgrade by using `roxctl` CLI, you can seamlessly transition the {rh-rhacs-first} services to use the stock {osp} SCCs, ensuring compatibility and optimal security configurations across Central and all secured clusters.
```

Comment:
Is _Central_ the full proper name? In some places in the assembly I see _Central database_. Wondering if this is the same or would a typical user (not me!) understand what we are referring to here?

Reply:
Yes, Central is the name of the component. No changes required.

Reply:
Technically, the full names for the two parts are `Central Services` and `Secured Cluster Services`. We often omit the `Services` part to shorten the name, but I trust you folks to decide whether we want to include the full name here or not.

Reply:
I would vote to leave it unchanged to maintain doc consistency.